roles/openshift_node_group/templates/node-config.yaml.j2

```diff
@@ -73,6 +73,11 @@ servingInfo:
   bindAddress: 0.0.0.0:10250
   bindNetwork: tcp4
   clientCA: client-ca.crt
+{% if osm_cluster_network_cidr is defined and not openshift_use_openshift_sdn | default(False) | bool %}
```

Comment:
er, no, I meant to say before, we should *always* set the variable. Just drop the conditional check entirely

Reply:
The variable we should use is openshift_cluster_network_cidr

Reply:
Sorry, misunderstood.